fastestimator/pipeline.py

```diff
@@ -184,7 +184,10 @@ def benchmark(self, mode: str = "train", epoch: int = 1, num_steps: int = 1000,
 
             data_len = len(loader.dataset.dataset)
             if self.batch_size:
-                log_interval = log_interval * self.batch_size
+                batch_size = self.batch_size.get_current_value(epoch) if isinstance(self.batch_size,
```

Comment:
need to separate the dictionary unpacking here so that a scheduled dictionary still works